packages/mdl-ripple/mdl-ripple.scss

```diff
@@ -26,14 +27,14 @@
   &__foreground,
   &__foreground-circle,
   &__background {
-    width: 100%;
-    height: 100%;
     position: absolute;
+    height: 100%;
+    width: 100%;
   }
 }
 
 .md-ripple__foreground {
-  &-active {
+  &-bounded-active {
   transition: transform 300ms cubic-bezier(0.157, 0.72, 0.386, 0.987);
```

Comment:
The reason for the compound modifier was just that there wasn't any CSS modification needed for the bounded case alone. It also turned out that the unbounded case didn't use class modifiers - it had to be done mostly via JS style manipulation.